decidim-accountability/lib/decidim/accountability/feature.rb

```diff
@@ -27,7 +27,16 @@
   end
 
   feature.register_stat :results_count, primary: true, priority: Decidim::StatsRegistry::HIGH_PRIORITY do |features, _start_at, _end_at|
```

Comment:
Is there any way to filter these features, depending on the participatory_space is private or public? I'm using the query below because I have not found any other way, but I think it's not the most optimal way .. @decidim/lot-core, any idea?

Reply:
@isaacmg410 The `HomeStats Presenter` only returns features from spaces matching the `public_spaces` scope:

https://github.com/decidim/decidim/blob/1d57472652e1697c38f90aabb5dbe7b5f64ba68d/decidim-core/app/presenters/decidim/home_stats_presenter.rb#L77-L81

You need to modify this scope in each space to return only those that are public. I would not change this result depending on the user, so I would only return those spaces that are public for everyone.